### DISCUSSION TITLE
GFN-FF with periodic boundary conditions

### DISCUSSION THREAD

**Discussion by manuelperezesc** [grimme-lab/xtb#786] [Ideas]
Is the use of periodic boundary conditions for molecular dynamics simulations already enabled in XTB? Is it enabled in DFTB+ instead?

**Comment by marcelmbn** [comment]
Should now (starting with `xtb-6.7.0`) be available in `xtb`.

**Reply by marcelmbn** [reply]
Ping also @Thomas3R on this.
